zeek-archiver/CMakeLists.txt

```diff
@@ -3,4 +3,4 @@ add_executable(zeek-archiver zeek-archiver.cc)
 set_target_properties(zeek-archiver PROPERTIES COMPILE_FLAGS
     "-std=c++17 -Wall -Wno-unused -Werror=vla")
 
-AddAuxInstallTarget(zeek-archiver)
+install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/zeek-archiver DESTINATION bin)
```

Comment:
The more cmake-y way to do this is to specify the `TARGET` instead of `PROGRAM`, since it already knows the path to the executable. That's not how we were doing it before though, so up to you on fixing it. Same below for `zeek-cut`.

Reply:
Thanks, I wondered! I switched it over.